### PR TITLE
Sidebar: Added

### PR DESCRIPTION
**Pull request by m26unkwn**
1. sidebar added
2. Auth added 
3. Protected Routes added

### CODE REVIEW

File: src/screens/sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import "./sidebar.css";
+
+export const Sidebar = () => {
+  return (
+    <div className='sidebar-wrapper '>
+      <div className='sidebar-content flex flex-col'>
+        <div className='sidebar-head flex ai-center jc-start'>
+          <NavLink to='/' className='flex'>
+            {/* <img src={Logo} alt='devhub_log' width='40px' height='40px' /> */}
```
Comment (dev-ashishsethi) [inline src/screens/sidebar/Sidebar.jsx:16]: don't comment code

**Review by dev-ashishsethi — APPROVED** [review]
looks good

**Review by Dev-AkshatGupta — APPROVED** [review]
This is all good a very small PR nothing to comment about

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,68 @@
...
+  const location = useLocation();
+
+  const locationFlag =
+    location.pathname === "/" ||
+    location.pathname === "/login" ||
+    location.pathname === "/signup";
   return (
-    <>
+    <div className={locationFlag ? "" : "main-wrapper"}>
+      {!locationFlag && <Sidebar />}
```
Comment (pankaj-prog) [inline src/App.js:26]: nice way of checking auth routes

File: src/App.js [hunk 3]
```diff
@@ -1,16 +1,68 @@
...
         <Route path='/' element={<Landing />} />
+
         <Route path='/login' element={<Login />} />
         <Route path='/signup' element={<Signup />} />
+        <Route
+          path='/home'
+          element={
+            <PrivateRoute>
+              <Home />
+            </PrivateRoute>
```
Comment (pankaj-prog) [inline src/App.js:37]: you don't need to repeat private route components for multiple, you can use outlet from router to do achieve the same thing

File: src/screens/labels/Lable.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/** @format */
+
+import React from "react";
+
+export const Lable = () => {
```
Comment (pankaj-prog) [inline src/screens/labels/Lable.jsx:5]: lable: typo mistake, it should be label

**Review by pankaj-prog — APPROVED** [review]

**Pull request merged**